uber/templates/jobs/index.html

```diff
@@ -20,24 +38,63 @@
         <input type="submit" value="I Confirm I Don't Need To Do Anything Else Here" />
         </form>
     {% endif %}
-    <br/><br/>
 {% endif %}
+</div>
+<div class="row text-center">
+	<div class="col-md-4">
+	    <h3>Setup Shifts ({{ setup|list_length }})</h3>	
+	</div>
+        <div class="col-md-4">
+            <h3>Teardown Shifts ({{teardown|list_length}})</h3>
+        </div>
+	<div class="col-md-4">
+	    <h3>Regular Shifts ({{normal|list_length}})</h3>
+	</div>
+
+</div>
+<div class="row text-center button_row">
+
+        <div class="col-md-4">
+	    <a class="btn btn-primary setup_button" href="form?id=None&location={{ location }}&type={{c.SETUP}}">Add Setup Shift</a>
+
+        </div>
+        <div class="col-md-4">
+            <a class="btn btn-primary teardown_button" href="form?id=None&location={{ location }}&type={{c.TEARDOWN}}">Add Teardown Shift</a>
 
-<div style="border:1px">
-    <h3>Setup Shifts</h3>
-    <a href="form?id=None&location={{ location }}&type={{ c.SETUP }}">Add a setup shift</a>
-    <ul>
-    {% for job in setup %}
+        </div>
+        <div class="col-md-4">
+            <a class="btn btn-primary" href="form?id=None&location={{ location }}">Add Shift</a>
+
+        </div>
+
+</div>
+{% comment %}
+<div class="row">
+        <div class="col-md-12">
+    {% for job in teardown %}
         <li>
             [<b>{{ job.start_time|datetime:"%I %p %a" }}</b> for an expected ~{{ job.duration }} hours]
             <a href="form?id={{ job.id }}">{{ job.name }}</a>:
             {{ job.shifts|length }}/{{ job.slots }} Staffers <a href="staffers_by_job?id={{ job.id }}">Signed Up</a>
         </li>
     {% endfor %}
-    </ul>
-    <br/>
+
+        </div>
 </div>
+{% endcomment %}
+
+
+
+<!-- Starting with calendar implementation -->
+<div class="row">
+	<div class="col-md-12">
+		<div id="shift_cal"></div>
+	</div>
+</div>
+<div class="row" style="margin-bottom:50px;"></div>
+
 
+<!--
```

Comment:
If code isn't being used anymore, feel free to remove it rather than commenting it out. Git and GitHub's tools allow us to easily inspect file history, so we always can find this code if we need to reference it.

By the way, it seems as though we don't need to return any of the variables this code was using. We'll want to clean those up from the .py function in order to avoid confusion.